common/source/docs/common-measuring-vibration.rst

```diff
@@ -18,7 +18,7 @@ find they are out-of-tolerance then follow the advice found on the :ref:`Vibrati
 
 Real-Time view in Ground Station
 --------------------------------
-
+nkf
```

Comment:
This "nkf" looks like it might have been an accidental addition..

Reply:
yep